### PR TITLE
[JENKINS-58999] Add documentation about job dsl and trigger mr comment trait

### PR DESCRIPTION
**Pull request by baymac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -424,28 +424,209 @@ The following behaviours apply to both `Multibranch Pipeline Jobs` and `Folder O
...
+              gitLabTagDiscovery() // discover tags
+            }
+          }
+        }
+        // "Traits" ("Behaviours" in the GUI) that are NOT "declarative-compatible"
+        // For some 'traits, we need to configure this stuff by hand until JobDSL handles it
+        // https://issues.jenkins.io/browse/JENKINS-45504
+        configure { node ->
+            def traits = node / navigators / 'io.jenkins.plugins.gitlabbranchsource.GitLabSCMNavigator' / traits
+            traits << 'io.jenkins.plugins.gitlabbranchsource.ForkMergeRequestDiscoveryTrait' {
```
[the remark this answers is not in the document]
Comment (baymac) [reply README.md]: Idk why but it doesn't override the traits block.

File: README.md [hunk 2]
```diff
@@ -424,28 +424,164 @@ The following behaviours apply to both `Multibranch Pipeline Jobs` and `Folder O
...
+            daysToKeep(10)
+            numToKeep(5)
+        }
+    }
+    // "Scan Organization Folder Triggers" : 1 day
+    // We need to configure this stuff by hand because JobDSL only allow 'periodic(int min)' for now
+    configure { 
+        it / triggers / 'com.cloudbees.hudson.plugins.folder.computed.PeriodicFolderTrigger' {
+            spec('H H * * *')
+            interval(86400000)
```
Comment (casz) [inline README.md]: Not sure this is necessary?
```
triggers {
  periodicFolderTrigger {
    interval('1d')
  }
}
```

Comment (baymac) [reply README.md]: It is there in the JCasC demo. If what you say works we can remove this.

Comment (casz) [reply README.md]: it works what I tested 👍

Comment (casz) [reply README.md]: oh... It works for child items...

**Pull request merged**